benches/bigint.rs

```diff
@@ -73,7 +73,16 @@ fn u512_mul(b: &mut Bencher) {
 	b.iter(|| {
 		let n = black_box(10000);
 		let one = black_box(U512::one());
-		(1..n).fold(one, |old, new| { old.overflowing_mul(U512::from(black_box(new))).0 })
+		(1..n).fold(one, |old, new| { old.overflowing_mul(U512::from(black_box(new | 1))).0 })
+	});
+}
+
+#[bench]
+fn u512_mul_small(b: &mut Bencher) {
+	b.iter(|| {
+		let n = black_box(10000);
+		let one = black_box(U512::one());
+		(1..153).fold(one, |old, new| { old.overflowing_mul(U512::from(10)).0 })
```

Comment:
so multiplying x10 128 times outputs the number which is multiple of 2^128, so 1/4 of 512-bit number (2 64bit words) contains only zeros?